### PR TITLE
Workflow to build and test on ARM64 for 5.x branch

### PR DESCRIPTION
**Pull request by fengyuentau**
Black list of tests can be found [here](https://github.com/opencv-cn-ci-bot/opencv-cn-pipelines/blob/master/precommit/ubuntu-20.04-arm64.groovy).

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: .github/workflows/PR-5.x-ARM64.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    - name: Merge opencv with a test branch
+      run: |
+        cd ${{ env.OPENCV_DOCKER_WORKDIR }}
+        git config --global --add safe.directory ${{ env.OPENCV_DOCKER_WORKDIR }}
+        git config user.email "opencv.ci"
+        git config user.name "opencv.ci"
+        git pull -v "https://github.com/${{ env.PR_AUTHOR_FORK }}" "${{ env.SOURCE_BRANCH_NAME }}"
+    - name: Clone opencv_contrib
+      run: git clone --single-branch --branch ${{ env.TARGET_BRANCH_NAME }} --depth 1 https://github.com/opencv/opencv_contrib.git /opencv_contrib
+    - name: Configure OpenCV Contrib
```
Comment (asenyaev) [inline .github/workflows/PR-5.x-ARM64.yaml:171]: Please add the same logic for merging OpenCV Contrib as it's [here](https://github.com/opencv/opencv/blob/4.x/.github/workflows/PR-4.x-U20.yaml#L169-L180).

File: .github/workflows/PR-5.x-ARM64.yaml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        git config --global --add safe.directory ${{ env.OPENCV_DOCKER_WORKDIR }}
+        git config user.email "opencv.ci"
+        git config user.name "opencv.ci"
+        git pull -v "https://github.com/${{ env.PR_AUTHOR_FORK }}" "${{ env.SOURCE_BRANCH_NAME }}"
+    - name: Clone opencv_extra
+      run: git clone --single-branch --branch ${{ env.TARGET_BRANCH_NAME }} https://github.com/opencv/opencv_extra.git /opencv_extra
+    - name: Merge opencv_extra with ${{ env.SOURCE_BRANCH_NAME }} branch
+      run: |
+        RET=$(git ls-remote --heads "https://github.com/${{ env.PR_AUTHOR }}/opencv_extra" "${{ env.SOURCE_BRANCH_NAME }}") || true
+        if [[ ! -z "$RET" ]]; then
```
Comment (asenyaev) [inline .github/workflows/PR-5.x-ARM64.yaml:54]: Let's rename `RET` to `OPENCV_EXTRA_FORK`?

**Review by asenyaev — CHANGES_REQUESTED** [review]

File: .github/workflows/PR-5.x-ARM64.yaml (hunk 1 above)
Comment (fengyuentau) [reply .github/workflows/PR-5.x-ARM64.yaml:171]: Done.

File: .github/workflows/PR-5.x-ARM64.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fengyuentau) [reply .github/workflows/PR-5.x-ARM64.yaml]: Yes, it is renamed to `OPENCV_EXTRA_FORK`.

**Review by asenyaev — APPROVED** [review]
Thanks @fengyuentau! :+1:

**Comment by fengyuentau** [comment]
It seems some failed unit tests are introduced from previous commits.

**Comment by asenyaev** [comment]
Looks like the [commit](https://github.com/opencv/opencv/commit/a99b4071a21c7029660b6a43d112d476e967f7ab) added some changes into 5.x.

**Comment by asmorkalov** [comment]
The related patch to OpenCV Extra has not been merged yet: https://github.com/opencv/opencv_extra/pull/889. Please wait a bit.

**Comment by fengyuentau** [comment]
Moved to a dedicated repo: https://github.com/opencv/ci-gha-workflow/pull/2

**Pull request closed without merging**